charts/ovirt-csi-driver-4.21.0-alpha1/values.yaml

```diff
@@ -0,0 +1,153 @@
+csiController:
+  csiAttacher:
+    containerSecurityContext:
+      privileged: true
+    image:
+      repository: olcne/csi-attacher
+      tag: v4.8.1
+    resources:
+      limits:
+        cpu: 160m
+        memory: 128Mi
+      requests:
+        cpu: 80m
+        memory: 64Mi
+  csiProvisioner:
+    containerSecurityContext:
+      privileged: true
+    image:
+      repository: olcne/csi-provisioner
+      tag: v5.2.0
+    resources:
+      limits:
+        cpu: 160m
+        memory: 128Mi
+      requests:
+        cpu: 80m
+        memory: 64Mi
+  csiResizer:
+    containerSecurityContext:
+      privileged: true
+    image:
+      repository: olcne/csi-resizer
+      tag: v1.13.2
+    resources:
+      limits:
+        cpu: 160m
+        memory: 128Mi
+      requests:
+        cpu: 80m
+        memory: 64Mi
+  livenessProbe:
+    image:
+      repository: olcne/livenessprobe
+      tag: v2.15.0
+    resources:
+      limits:
+        cpu: 160m
+        memory: 128Mi
+      requests:
+        cpu: 80m
+        memory: 64Mi
+  ovirtController:
+    name: ovirt-csi-controller-plugin
+    containerSecurityContext:
+      privileged: true
+    env:
+      csiEndpoint: unix:///csi-socket/csi.sock
+      ovirtConfig: /tmp/config/ovirt-config.yaml
+    image:
+      repository: olcne/ovirt-csi-driver
+      tag: v4.21.0
```

Comment:
Will this tag have "-alpha1" added to it?

Reply:
no, image tag is just  v4.21.0